library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/DatePickerDialog.kt

```diff
@@ -0,0 +1,51 @@
+package com.epmedu.animeal.foundation.dialog
+
+import android.annotation.SuppressLint
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.epmedu.animeal.resources.R
+import com.vanpra.composematerialdialogs.MaterialDialog
+import com.vanpra.composematerialdialogs.datetime.date.datepicker
+import com.vanpra.composematerialdialogs.rememberMaterialDialogState
+import java.time.LocalDate
+
+@Composable
+fun DatePickerDialog(
+    initialDate: LocalDate,
+    shouldShowDialog: MutableState<Boolean>,
```

Comment:
I guess it's not very good idea to pass the mutable state here, you could use a callback and a simple value instead and handle the mutable state outside

Reply:
Not mine, but fixed